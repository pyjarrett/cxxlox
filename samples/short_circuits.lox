print 0 and 10;
print 10 and 0;
print 5 or 20;
print 20 or 5;