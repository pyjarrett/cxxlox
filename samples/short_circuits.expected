10
0
5
20
